hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -152,6 +153,7 @@ public static HoodieWriteConfig getHoodieClientConfig(Configuration conf) {
                     // actually Flink cleaning is always with parallelism 1 now
                     .withCleanerParallelism(20)
                     .archiveCommitsWith(conf.getInteger(FlinkOptions.ARCHIVE_MIN_COMMITS), conf.getInteger(FlinkOptions.ARCHIVE_MAX_COMMITS))
+                    .withCleanerPolicy(HoodieCleaningPolicy.KEEP_LATEST_COMMITS)
                     .build())
```

Comment:
Isn't `HoodieCleaningPolicy.KEEP_LATEST_COMMITS` the default value ?

Reply:
build() method get policy use compactionConfig.getString(CLEANER_POLICY_PROP),this method doesn't return default value, but null. if not set then will return null.

Reply:
The `HoodieCompactionConfig` would `setDefaults` first before build.